### DISCUSSION TITLE
Are there scenarios that we no longer need to run?

### DISCUSSION THREAD

**Discussion by JaneSullivan-NOAA** [afsc-assessments/AK-projection-simulation#15] [General]
Below is a summary of the seven standard scenarios run. Based on discussion with @jimianelli and @Cole-Monnahan-NOAA, its possible that some of these can be removed. Do we need to talk with Council staff about this or is it a GPT/SSC process?

Five of the seven standard scenarios will be used in an Environmental Assessment prepared in conjunction with the final SAFE. These five scenarios, which are designed to provide a range of harvest alternatives that are likely to bracket the final TAC for 2022, are as follow (“max FABC” refers to the maximum permissible value of FABC under Amendment 56):

**Scenario 1:**  In all future years, F is set equal to max FABC. (Rationale:  Historically, TAC has been constrained by ABC, so this scenario provides a likely upper limit on future TACs.)
**Scenario 2:**  In 2022 and 2023, F is set equal to a constant fraction of max FABC, where this fraction is equal to the ratio of the realized catches in 2018-2020 to the ABC recommended in the assessment for each of those years. For the remainder of the future years, maximum permissible ABC is used. (Rationale:  In many fisheries the ABC is routinely not fully utilized, so assuming an average ratio of F will yield more realistic projections.)
**Scenario 3:**  In all future years, F is set equal to 50% of max FABC. (Rationale:  This scenario provides a likely lower bound on FABC that still allows future harvest rates to be adjusted downward when stocks fall below reference levels.)
**Scenario 4:**  In all future years, F is set equal to the 2016-2020 average F. (Rationale:  For some stocks, TAC can be well below ABC, and recent average F may provide a better indicator of FTAC than FABC.)
**Scenario 5:**  In all future years, F is set equal to zero. (Rationale:  In extreme cases, TAC may be set at a level close to zero.)

Two other scenarios are needed to satisfy the MSFCMA’s requirement to determine whether a stock is currently in an overfished condition or is approaching an overfished condition.  These two scenarios are as follow (for Tier 3 stocks, the MSY level is defined as B35%):
**Scenario 6:**  In all future years, F is set equal to FOFL. (Rationale:  This scenario determines whether a stock is overfished. If the stock is expected to be 1) above its MSY level in 2021 or 2) above ½ of its MSY level in 2021 and above its MSY level in 2031 under this scenario, then the stock is not overfished.)
**Scenario 7:**  In 2022 and 2023, F is set equal to max FABC, and in all subsequent years F is set equal to FOFL. (Rationale:  This scenario determines whether a stock is approaching an overfished condition. If the stock is 1) above its MSY level in 2023 or 2) above ½ of its MSY level in 2023 and expected to be above its MSY level in 2033 under this scenario, then the stock is not approaching an overfished condition.)

**Comment by zandjyo** [comment]
I think scenario 2 differs among assessments.

**Reply by zandjyo** [reply]
Actually a number of the scenarios differ for EBS cod:

**Scenario 1:** In all future years, F is set equal to max FABC. (Rationale: Historically, TAC has been
constrained by ABC, so this scenario provides a likely upper limit on future TACs.)
**Scenario 2:** In all future years, F is set equal to a constant fraction (“author’s F”) of max FABC,
where this fraction is equal to the ratio of the FABC value for 2023 recommended in the assessment
to the max FABC for 2023, and where catches for 2023 and 2024 are estimated at their most likely
values given the 2023 and 2024 recommended ABCs under this scenario. (Rationale: When FABC
is set at a value below max FABC, it is often set at the value recommended in the stock assessment;
also, catch tends not to equal ABC exactly.)
**Scenario 3:** In all future years, F is set equal to the 2017-2021 average F. (Rationale: For some
stocks, TAC can be well below ABC, and recent average F may provide a better indicator of FTAC
than FABC.)
**Scenario 4:** In all future years, the upper bound on FABC is set at F60%. (Rationale: This scenario
provides a likely lower bound on FABC that still allows future harvest rates to be adjusted
downward when stocks fall below reference levels.)
**Scenario 5**: In all future years, F is set equal to zero. (Rationale: In extreme cases, TAC may be
set at a level close to zero.)
Two other scenarios are needed to satisfy the MSFCMA’s requirement to determine whether a stock is
currently in an overfished condition or is approaching an overfished condition. These two scenarios are as
follow (for Tier 3 stocks, the MSY level is defined as B35%):
**Scenario 6:** In all future years, F is set equal to FOFL. (Rationale: This scenario determines
whether a stock is overfished. If the stock is 1) above its MSY level in 2022 or 2) above 1/2 of its
MSY level in 2022 and expected to be above its MSY level in 2032 under this scenario, then the
stock is not overfished.)
**Scenario 7:** In 2023, F is set equal to max FABC, and in all subsequent years, F is set equal to FOFL.
(Rationale: This scenario determines whether a stock is approaching an overfished condition. If
the stock is 1) above its MSY level in 2024 or 2) above 1/2 of its MSY level in 2023 and
expected to be above its MSY level in 2034 under this scenario, then the stock is not approaching
an overfished condition.)

**Reply by sandra-lowe** [reply]
Last year when reviewing stock assessments, I noticed that the scenarios differed among assessments, the years listed differed, and the order of the scenarios were swapped (3 and 4) in some assessments. Scenario 4 is listed as 50% (original value) in some assessments, and as 60% in others (updated value). I went through the original 2005 document:

[Projection Model 2005.docx](https://github.com/afsc-assessments/spmR/files/11368932/Projection.Model.2005.docx)
and compared with Grant's latest EBS cod assessments. Assumption is that Grant was always right! I updated the scenarios as best I could, leaving out years and replacing with "assessment yr", assessment yr+1", etc. in the following file:

[Projection scenarios updated text.docx](https://github.com/afsc-assessments/spmR/files/11368942/Projection.scenarios.updated.text.docx)

Is this helpful? We need to straighten out the scenarios and have them in the Guidelines or at least in an SOP for proj?

**Reply by Cole-Monnahan-NOAA** [reply]
Hi Sandra yes we plan on making the scenarios consistent and giving some guidelines. Jane and I plan to do that this year. Thanks for sharing!

**Comment by jimianelli** [comment]
This isn't technically a council issue (except for 1 and 2 mainly), it's
more MSFCMA related. Scenarios 3-5 were part of AKRO Env Assessment coming
from the PSEIS. I'd root for getting rid of Scenario 3, I find 4 and 5
useful, and would love to get rid of 6 and 7 but probably can't...

On Thu, Mar 23, 2023 at 3:47 PM Jane Sullivan ***@***.***>
wrote:

> Below is a summary of the seven standard scenarios run. Based on
> discussion with @jimianelli <https://github.com/jimianelli> and
> @Cole-Monnahan-NOAA <https://github.com/Cole-Monnahan-NOAA>, its possible
> that some of the can be removed. Do we need to talk with Council staff
> about this or is it a GPT/SSC process?
>
> Five of the seven standard scenarios will be used in an Environmental
> Assessment prepared in conjunction with the final SAFE. These five
> scenarios, which are designed to provide a range of harvest alternatives
> that are likely to bracket the final TAC for 2022, are as follow (“max
> FABC” refers to the maximum permissible value of FABC under Amendment 56):
>
> *Scenario 1:* In all future years, F is set equal to max FABC.
> (Rationale: Historically, TAC has been constrained by ABC, so this scenario
> provides a likely upper limit on future TACs.)
> *Scenario 2:* In 2022 and 2023, F is set equal to a constant fraction of
> max FABC, where this fraction is equal to the ratio of the realized catches
> in 2018-2020 to the ABC recommended in the assessment for each of those
> years. For the remainder of the future years, maximum permissible ABC is
> used. (Rationale: In many fisheries the ABC is routinely not fully
> utilized, so assuming an average ratio of F will yield more realistic
> projections.)
> *Scenario 3:* In all future years, F is set equal to 50% of max FABC.
> (Rationale: This scenario provides a likely lower bound on FABC that still
> allows future harvest rates to be adjusted downward when stocks fall below
> reference levels.)
> *Scenario 4:* In all future years, F is set equal to the 2016-2020
> average F. (Rationale: For some stocks, TAC can be well below ABC, and
> recent average F may provide a better indicator of FTAC than FABC.)
> *Scenario 5:* In all future years, F is set equal to zero. (Rationale: In
> extreme cases, TAC may be set at a level close to zero.)
>
> Two other scenarios are needed to satisfy the MSFCMA’s requirement to
> determine whether a stock is currently in an overfished condition or is
> approaching an overfished condition. These two scenarios are as follow (for
> Tier 3 stocks, the MSY level is defined as B35%):
> *Scenario 6:* In all future years, F is set equal to FOFL. (Rationale:
> This scenario determines whether a stock is overfished. If the stock is
> expected to be 1) above its MSY level in 2021 or 2) above ½ of its MSY
> level in 2021 and above its MSY level in 2031 under this scenario, then the
> stock is not overfished.)
> *Scenario 7:* In 2022 and 2023, F is set equal to max FABC, and in all
> subsequent years F is set equal to FOFL. (Rationale: This scenario
> determines whether a stock is approaching an overfished condition. If the
> stock is 1) above its MSY level in 2023 or 2) above ½ of its MSY level in
> 2023 and expected to be above its MSY level in 2033 under this scenario,
> then the stock is not approaching an overfished condition.)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/afsc-assessments/spmR/discussions/15>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAUW7YXR5SMVJQDBZCJOCY3W5SLDJANCNFSM6AAAAAAWFTFGK4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Jim Ianelli

**Reply by sandra-lowe** [reply]
Yes, the PSEIS contains the 5 scenarios. We need to keep them in to stay compliant with NEPA. Ironic timing that we are now being sued on being out of compliance because the 5 scenarios were analyzed under different environmental conditions. Status determinations, Scenarios 6 and 7 are a NMFS requirement we can't drop.

**Comment by jimianelli** [comment]
there was some re-ordering of the scenarios in the main code (forget why)
that some folks didn't adopt. The descriptions should remain correct within
the assessments.

The year listings may have lapsed in some updates (sometimes hard to get
all the places where things changed...).

Regarding a single SOP, completely agree and that is the plan. I don't
think there's a big need to revisit all the different cases of history
where different things might have been done, just to set things straight
(and simple/easy!).

Cheers,
Jim

On Mon, May 1, 2023 at 6:11 PM sandra-lowe ***@***.***> wrote:

> Last year when reviewing stock assessments, I noticed that the scenarios
> differed among assessments, the years listed differed, and the order of the
> scenarios were swapped (3 and 4) in some assessments. Scenario 4 is listed
> as 50% (original value) in some assessments, and as 60% in others (updated
> value). I went through the original 2005 document:
>
> Projection Model 2005.docx
> <https://github.com/afsc-assessments/spmR/files/11368932/Projection.Model.2005.docx>
> and compared with Grant's latest EBS cod assessments. Assumption is that
> Grant was always right! I updated the scenarios as best I could, leaving
> out years and replacing with "assessment yr", assessment yr+1", etc. in the
> following file:
>
> Projection scenarios updated text.docx
> <https://github.com/afsc-assessments/spmR/files/11368942/Projection.scenarios.updated.text.docx>
>
> Is this helpful? We need to straighten out the scenarios and have them in
> the Guidelines or at least in an SOP for proj?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/afsc-assessments/spmR/discussions/15#discussioncomment-5776182>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAUW7YSXPOEWOEDYJ2L2VFLXEBNKZANCNFSM6AAAAAAWFTFGK4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Jim Ianelli

**Reply by JaneSullivan-NOAA** [reply]
hey @jimianelli i'm reviving this thread. just fyi i reached out to gc molly for guidance on which scenarios are legally required...

**Comment by jimianelli** [comment]
Ahh sure

Jim Ianelli
